IdentityServer/v6/docs/content/quickstarts/4_ef.md

```diff
@@ -4,117 +4,193 @@ date: 2020-09-10T08:22:12+02:00
+with IdentityServer's models - when the models are changed in a new release,
+corresponding changes are made to the entities. As you use IdentityServer and
+upgrade over time, you are responsible for your database schema and changes
+necessary to that schema.
 
-```cs
-var migrationsAssembly = typeof(Startup).GetTypeInfo().Assembly.GetName().Name;
-const string connectionString = @"Data Source=Duende.IdentityServer.Quickstart.EntityFramework.db";
+One approach for managing those changes is to use [EF
+migrations](https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/index),
+which is what this quickstart will use. If migrations are not your preference,
+then you can manage the schema changes in any way you see fit. 
 
-services.AddIdentityServer()
-    .AddTestUsers(TestUsers.Users)
-    .AddConfigurationStore(options =>
-    {
-        options.ConfigureDbContext = b => b.UseSqlite(connectionString,
-            sql => sql.MigrationsAssembly(migrationsAssembly));
-    })
-    .AddOperationalStore(options =>
-    {
-        options.ConfigureDbContext = b => b.UseSqlite(connectionString,
-            sql => sql.MigrationsAssembly(migrationsAssembly));
-    });
-```
+{{% notice note %}}
 
-You might need these namespaces added to the file:
+You can find the latest SQL scripts for SqlServer in our EF
+[repository](https://github.com/DuendeSoftware/IdentityServer/tree/main/migrations/IdentityServerDb/Migrations).
 
-```cs
-using Microsoft.EntityFrameworkCore;
-using System.Reflection;
+{{% /notice %}}
+
+### Adding Migrations
+To create migrations, you will need to install the Entity Framework Core CLI
+tool on your machine and the *Microsoft.EntityFrameworkCore.Design* nuget
+package in IdentityServer. Run the following commands from the *IdentityServer*
+directory.
+
+```console
+dotnet tool install --global dotnet-ef
+dotnet add package Microsoft.EntityFrameworkCore.Design
 ```
 
-Because we are using EF migrations in this quickstart, the call to *MigrationsAssembly(...)* is used to inform Entity Framework that the host project will contain the migrations code.
-This is necessary since the host project is in a different assembly than the one that contains the *DbContext* classes.
+### Handle Excepted Exception
+The Entity Framework CLI internally start up the host in order to read the
+database configuration. However, it does not want to actually run the host after
+it has read the config, so it immediately throws a *StopTheHostException*. Since
+this exception is expected and not a fatal error, update the error logging code
+in *IdentityServer\Program.cs* as follows:
+```csharp
+catch (Exception ex)
+{
+    if (ex.GetType().Name == "StopTheHostException")
+    {
+        throw;
+    }
 
-### Adding Migrations
-Once the IdentityServer has been configured to use Entity Framework, we’ll need to generate some migrations.
+    Log.Fatal(ex, "Unhandled exception");
+}
```

Comment:
I didn't look at the code in the sample repo (yet), but is this logic right? I would expect to `return` on the STH exception, but re-throw for others? I might have it backwards, tho.